libraries/AP_HAL_Linux/VideoIn.cpp

```diff
@@ -41,6 +42,29 @@ extern const AP_HAL::HAL& hal;
 
 using namespace Linux;
 
+int VideoIn::get_fd(void)
+{
+    return _fd;
+}
+
```

Comment:
I was using in the `OpticalFlow_Onboard.cpp` but after some changes I forgot to delete this function since its not used anywhere, nice catch (: